static/app/components/dataExport.tsx

```diff
@@ -44,6 +44,7 @@ function DataExport({
     if (inProgress) {
       setInProgress(false);
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I would just make this commit without the precommit checks, since we probably don't want to actually ignore these right away